packages/design-system/package.json

```diff
@@ -125,6 +130,10 @@
     "tailwindcss-animate": "^1.0.7",
     "usehooks-ts": "^3.1.0",
     "viem": "^2.16.1",
-    "wagmi": "^2.12.17"
+    "wagmi": "^2.12.17",
+    "form": "^0.2.5",
+    "add": "^2.0.6",
+    "@radix-ui/react-slot": "^1.1.0",
+    "@radix-ui/react-form": "^0.1.0"
```

Comment:
Do we need all those new dependencies? I'm noting that `zod` is duplicated here

Reply:
you are right, this will close , because we move to different path for validation